packages/request-nanostores/virtual.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '@it-astro:request-nanostores' {
+	import { ReadableAtom } from 'nanostores';
```

Comment:
shouldn't that be WritableAtom?

Reply:
You can make a readable atom that is an object and mutate the object throughout the request. On the client it will retain those changes.

Think of it like this:
```ts
// A writable atom
let writable = {};

// A readable atom
const readable = {};

// you can still do
readable.foo = 'bar';
```